tests/main.cpp

```diff
@@ -13,20 +13,46 @@ using namespace blaze::iterative;
 // Distributed under the Boost Software License, Version 1.0. (See accompanying
 // file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
+template< typename MT, bool SO>
+void copyStrictlyLowerPart( DynamicMatrix<MT,SO>& src, DynamicMatrix<MT,SO>& dst )
```

Comment:
Can this reuse the function defined above (in `include/solvers/PreconditionCG.hpp`)?